stdlib/public/core/Unicode.swift

```diff
@@ -847,19 +742,19 @@ extension UTF16 {
     source source: UnsafeMutablePointer<T>,
     destination: UnsafeMutablePointer<U>,
     count: Int
-  ) {
-    if strideof(T.self) == strideof(U.self) {
-      _memcpy(
-        dest: UnsafeMutablePointer(destination),
-        src: UnsafeMutablePointer(source),
-        size: UInt(count) * UInt(strideof(U.self)))
-    }
-    else {
-      for i in 0..<count {
-        let u16 = T._toUTF16CodeUnit((source + i).pointee)
-        (destination + i).pointee = U._fromUTF16CodeUnit(u16)
+    ) {
+      if strideof(T.self) == strideof(U.self) {
+        _memcpy(
+          dest: UnsafeMutablePointer(destination),
+          src: UnsafeMutablePointer(source),
+          size: UInt(count) * UInt(strideof(U.self)))
+      }
+      else {
+        for i in 0..<count {
+          let u16 = T._toUTF16CodeUnit((source + i).pointee)
+          (destination + i).pointee = U._fromUTF16CodeUnit(u16)
+        }
```

Comment:
Please fix the indentation here (-2 spaces).  In general it's a good idea to check your diffs for whitespace changes, which only introduce churn.